projects/storefrontstyles/scss/theme/sparta/_fonts.scss

```diff
@@ -1 +1,5 @@
-@import url('https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700');
+// TODO: Replace `$font-url` with swap url in 5.0 to use swap strategy by default.
+// $font-url: 'https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700&display=swap';
+$font-url: 'https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700';
+
+@import url('#{$fontUrl}');
```

Comment:
```
ModuleBuildError: Module build failed (from ./node_modules/@angular-devkit/build-angular/node_modules/sass-loader/dist/cjs.js):
SassError: Undefined variable.
  ╷
5 │ @import url('#{$fontUrl}');
```

Seems that we are trying to import wrong variable. Shouldn't be `$font-url` instead?